### PR TITLE
feat: add missing sync action for StandalonePrice entity

### PR DESCRIPTION
**Pull request by ragafus**
#### Summary
This PR adds support for StandalonePrice `setKey`, `setValidFrom`, `setValidUntil`, `setValidFromAndUntil` and `changeActive` actions.

#### Todo
Staged `changeValue` update action needed for https://commercetools.atlassian.net/browse/PRC-2676 will be introduced with a follow-up ticket

- Tests
  - [ ] Unit
  - [ ] Integration
  - [ ] Acceptance
- [ ] Documentation
- [ ] `Type` label for the PR

### CODE REVIEW

**Review by ARRIOLALEO — APPROVED** [review]

**Review by dogayuksel — APPROVED** [review]

**Review by markus-azer — APPROVED** [review]
Looks really pretty good 👍 💯 , just ci complaining about more testing, and its kinda hard to follow the added tests cause of the added indentation.

**Review by ARRIOLALEO — APPROVED** [review]
